k2/csrc/ragged.h

```diff
@@ -338,6 +338,53 @@ struct Ragged {
 template <typename T>
 std::ostream &operator<<(std::ostream &stream, const Ragged<T> &r);
 
+/*
+  Stack a list of Ragged arrays to create a Ragged array with one more axis.
+  Similar to TF/PyTorch's Stack.  The result will have Dim0 == num_srcs.  All
+  the source Ragged arrays' shapes must have the same NumAxes().
+
+     @param [in] axis   The new axis whose dimension will equal num_srcs.
+                        CAUTION: only axis == 0 is supported right now.
```

Comment:
actually the implementation seems to support axis == 1 right now so this comment isnt right currently

Reply:
Oh yes, we have added `Transpose` in it after the first implementation.